ansible/roles/namespace-activator-scaled-objects/defaults/main.yml

```
@@ -36,5 +36,7 @@ deployments_for_scaled_objects:
   - name: tldraw-worker-deployment
   - name: tldraw-client-deployment
   - name: tldraw-valkey-node
+  - name: schulconnex-group-provisioning-deployment
+  - name: schulconnex-license-provisioning-deployment
     kind: StatefulSet
     replicaCount: "{{TLDRAW_VALKEY_REPLICAS}}"
```

You will messed up with valkey for tldraw and schulconnex-license-provisioning-deployment how you hade written it
```suggestion
  - name: schulconnex-group-provisioning-deployment
  - name: schulconnex-license-provisioning-deployment
  - name: tldraw-valkey-node
    kind: StatefulSet
    replicaCount: "{{TLDRAW_VALKEY_REPLICAS}}"
```